src/main/java/org/sagebionetworks/bridge/exporter/helper/BridgeHelper.java

```diff
@@ -62,11 +64,21 @@ public void updateRecordExporterStatus(List<String> recordIds, RecordExportStatu
         setCredentials(credentials);
 
         // then update status
-        RecordExportStatusRequest request = new RecordExportStatusRequest(recordIds, status);
-        sessionHelper(() -> {
-            session.getWorkerClient().updateRecordExporterStatus(request);
-            return null;
+        // breaking down the list into batches whenever the list size exceeds the batch size
+        List<List<String>> batches = Lists.partition(recordIds, MAX_BATCH_SIZE);
+        batches.forEach(batch-> {
+            RecordExportStatusRequest request = new RecordExportStatusRequest(batch, status);
+            sessionHelper(() -> {
+                session.getWorkerClient().updateRecordExporterStatus(request);
+                return null;
+            });
+            try {
+                TimeUnit.SECONDS.sleep(1);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```

Comment:
Minor: Instead of e.printStackTrace(), use LOG.warn(). This will ensure our output goes to the logs.